app.js

```diff
@@ -1,10 +1,53 @@
 const WebSocket = require('ws');
 require('dotenv').load();
-var exp = require('./index.js');
 
+// when socket opens, notify channel
+function onOpen (soc) {
+    console.log('socket opened');
+    var msg = {
+        "id": 1,
+        "type": "message",
+        "channel": "C1BBWJ7PF",
+        "text": "WhoopBot connected to WebSocket"
+    };
+    
+    if (soc.readyState == 1) {
+        soc.send(JSON.stringify(msg));
+    };
```

Comment:
Use `===` in JavaScript. `==` is a weird operator here.

Is `readyState` actually going to equal `1`? If it's a truthy/falsey value, you should use it like a boolean: `if (soc.readyState) {` 

Have you seen `soc.readyState` being false? What does it mean when it's not ready?